docs/toc.yml

```diff
@@ -267,6 +267,13 @@
       href: core/docker/build-container.md
     - name: Container Tools in Visual Studio
       href: /visualstudio/containers/overview
+  - name: Diagnostics
```

Comment:
Our site has changed the way these parent nodes operate. If the parent node, such as this one, has an article attached to it, it gets replicated as a child automatically. This causes the TOC to have two entries, one expandable and one clickable. They used to be expand+click but not anymore. The better way to do this is to create a true child with an alternative title:

```yml
  - name: Diagnostics
    items:
    - name: Overview
      href: core/diagnostics/index.md
    - name: Managed Debuggers
      href: core/diagnostics/managed-debuggers.md
    - name: Logging and Tracing
      href: core/diagnostics/logging-tracing.md
```